docs/programming.rst

```diff
@@ -29,7 +29,7 @@ The core API contains the central components of PSI/J. The core API classes are
 :class:`~psij.job_executor.JobExecutor`
     The base class for a concrete method of executing a job, such as Slurm, or PBS.
 
-:class:`~psij.launchers.launcher.Launcher`
+:class:`~psij.launcher.Launcher`
```

Comment:
Looks like this file needs updating.

Reply:
Thanks for spotting it. I somehow missed it when adding files to the commit.